src/deployconfig.jl

```diff
@@ -492,6 +538,8 @@ function auto_detect_deploy_system()
         return Travis()
     elseif haskey(ENV, "GITHUB_REPOSITORY")
         return GitHubActions()
+    elseif haskey(ENV, "DOCUMENTER_KEY_PATH")
+        return SimpleSSHConfig()
```

Comment:
I like this. It allows you to do a manual deployment without changing the `make.jl` script. I wonder if we should still default to `~/.ssh/id_rsa` if `isempty(DOCUMENTER_KEY_PATH)`? Then you can just run like this locally:

```
DOCUMENTER_KEY_PATH= julia --project=docs/ docs/make.jl
```

Reply:
Actually, in light of what we were discussing before with this always pushing, that seems rather dangerous.  In fact, now I'm even wondering if it's wise to do even this, could result in some users pushing when they don't intend to.

Reply:
Hmm. You are probably right. However, I feel that it is a nice workflow. Maybe one option would be to use a separate variable to trigger the automatic deployment, e.g. `DOCUMENTER_DEPLOY=local` and have it match `local` exactly?